app/views/article/record/_section_fields.html.erb

```diff
@@ -0,0 +1,14 @@
+<%
+doc_presenter = presenter(document)
+fields.each do |field_name, label|
+  if doc_presenter.respond_to?(field_name)
+    value = doc_presenter.send(field_name)
+  else
+    value = document[field_name.to_s]
+  end
+ -%>
+  <% if value.present? %>
+    <dt><%= label -%></dt>
+    <dd><%= safe_join(Array.wrap(value)) -%></dd>
```

Comment:
This is already being `safe_joined` in the presenter I think?  Do we need it both places?